sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/BeamCalciteTable.java

```diff
@@ -40,7 +40,7 @@
 import org.apache.calcite.schema.TranslatableTable;
 
 /** Adapter from {@link BeamSqlTable} to a calcite Table. */
-class BeamCalciteTable extends AbstractQueryableTable
```

Comment:
`BeamCalciteTable` is a private wrapper for use by `BeamCalciteSchema`, you shouldn't need to access it directly.